Assets/Scripts/Components/Interactable/GrabInteractableSocket.cs

```diff
@@ -184,18 +242,13 @@ private void OnTriggerEnter(Collider other) // 호버 감지용임 헷갈리지
         
         // Set Hologram
         SetHologramMesh(other);
-        SetHologramSize(other);
         
         _hologram.SetActive(true);
         
     }
-
-
-    
     
     private void OnTriggerExit(Collider other)
     {
-
         GrabInteractable __otherGrabInteractable = other.GetComponent<GrabInteractable>();
```

Comment:
특별히 지역변수인 걸 명시하지 않습니다~
```suggestion
        GrabInteractable otherGrabInteractable = other.GetComponent<GrabInteractable>();
```